Assets/DeathMario.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public sealed class DeathMario : DeathBase
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+
+    }
+
+    // Update is called once per frame
+    protected new void Update()
+    {
+        base.Update();
+    }
+
+
+    public override void Die()
+    {
+        if (_isDead == false)
+        {
+            // 演出
+            var rb = this.GetComponent<Rigidbody2D>();
+
+            if (rb != null) rb.velocity += Vector2.up * 3f;
```

Comment:
```suggestion
            const float _deathJumpPower = 3f;
```
　固定値を使う場合は、ハードコーディングではなく、クラスの先頭にconst属性をつけてまとめていただくか、　別にStaticなConstクラスを作ってもらい、一括して読み込めるようにしてほしい